From f99b2af70809db1221fa9a8a0c5a533a8f56045c Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Wed, 15 Aug 2018 17:06:02 +0800
Subject: [PATCH 4475/5242] MLK-19227-1: arm64:dts:imx8qm: Add HDMI/DP HPD
 interrupter

commit  64ad863c50368698760f0799191bec8c357dee94 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add HDMI/DP HPD interrupter.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
(cherry picked from commit 36ae1b30f8a2128128698f6496b28549a90721ee)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-device.dtsi      |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index 3a7e68f..954f310 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -1776,7 +1776,9 @@
 		#size-cells = <0>;
 		reg = <0x0 0x56268000 0x0 0x100000>, /* HDP Controller */
 				<0x0 0x56261000 0x0 0x1000>; /* HDP SubSystem CSR  */
-		interrupts = <10 IRQ_TYPE_LEVEL_HIGH>;
+		interrupts = <10 IRQ_TYPE_LEVEL_HIGH>,
+						<13 IRQ_TYPE_LEVEL_HIGH>;
+		interrupt-names = "plug_in", "plug_out";
 		interrupt-parent = <&irqsteer_hdmi>;
 		status = "disabled";
 		clocks = <&clk IMX8QM_HDMI_DIG_PLL_CLK>,
-- 
1.7.9.5

